pkg/ops/iso.go

```diff
@@ -5,20 +5,110 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
 
 	"github.com/kairos-io/AuroraBoot/internal"
+	"github.com/kairos-io/AuroraBoot/pkg/constants"
 	"github.com/kairos-io/AuroraBoot/pkg/schema"
+	"github.com/kairos-io/AuroraBoot/pkg/utils"
 	"github.com/otiai10/copy"
+	"github.com/twpayne/go-vfs/v4"
 
-	enkiaction "github.com/kairos-io/enki/pkg/action"
-	enkiconfig "github.com/kairos-io/enki/pkg/config"
-	enkiconstants "github.com/kairos-io/enki/pkg/constants"
-	enkitypes "github.com/kairos-io/enki/pkg/types"
-	v1 "github.com/kairos-io/kairos-agent/v2/pkg/types/v1"
+	"github.com/kairos-io/kairos-agent/v2/pkg/cloudinit"
+	agentconfig "github.com/kairos-io/kairos-agent/v2/pkg/config"
+	"github.com/kairos-io/kairos-agent/v2/pkg/elemental"
+	"github.com/kairos-io/kairos-agent/v2/pkg/http"
+	v1types "github.com/kairos-io/kairos-agent/v2/pkg/types/v1"
 	sdkTypes "github.com/kairos-io/kairos-sdk/types"
-	"github.com/kairos-io/kairos-sdk/utils"
+	sdkutils "github.com/kairos-io/kairos-sdk/utils"
+	"github.com/sanity-io/litter"
 )
 
+type LiveISO struct {
+	RootFS             []*v1types.ImageSource `yaml:"rootfs,omitempty" mapstructure:"rootfs"`
+	UEFI               []*v1types.ImageSource `yaml:"uefi,omitempty" mapstructure:"uefi"`
+	Image              []*v1types.ImageSource `yaml:"image,omitempty" mapstructure:"image"`
+	Label              string                 `yaml:"label,omitempty" mapstructure:"label"`
+	GrubEntry          string                 `yaml:"grub-entry-name,omitempty" mapstructure:"grub-entry-name"`
+	BootloaderInRootFs bool                   `yaml:"bootloader-in-rootfs" mapstructure:"bootloader-in-rootfs"`
+}
+
+// BuildConfig represents the config we need for building isos, raw images, artifacts
+type BuildConfig struct {
```

Comment:
I'm reluctant to create a "config" package in auroraboot because I'm hoping we can get rid of all these "config" structs that behave like "store everything" objects. That's why I put all the structs next to where they are used, to make it simpler to refactor.